7200_Final_TechIndex/src/test/scala/ReadDataSpec.scala

```diff
@@ -0,0 +1,41 @@
+import java.io.File
+import java.time.LocalDate
+import java.time.format.DateTimeFormatter
+
+import org.scalatest.{FlatSpec, Matchers}
+
+class ReadDataSpec extends FlatSpec with Matchers {
+
+  behavior of "readFile"
+  it should "read the file get the date and price" in {
+
+    val appleFile =  new File("data/stocks/AAPL.csv")
+    val appleData = new ReadData().readFile(appleFile)
+    appleData(0)._1.getClass.getSimpleName shouldBe("LocalDate")
+    appleData(0)._2.getClass.getSimpleName shouldBe("double")
+
+  }
+
+
+  behavior of "trimToRegion"
+  it should "get the data in the time we want" in {
+    val readData = new ReadData()
+    val appleFilename = "data/stocks/AAPL.csv"
```

Comment:
You don't need to use all of the data to test, just use one specific file to test the function